pywr/recorders/recorders.py

```diff
@@ -473,6 +473,33 @@ def setup(self):
                         entry[s.name] = i
                     entry.append()
                 tbl.flush()
+            elif any([s.slice for s in self.model.scenarios.scenarios]):
+                # Slices are only applied in a model run if there are no user combinations
+                description = {
+                    "name": tables.StringCol(1024),
+                    "slice_start": tables.Int64Col(),
+                    "slice_stop": tables.Int64Col(),
+                    "slice_step": tables.Int64Col(),
+                }
+                tbl = self.h5store.file.create_table(
+                    group_name,
+                    "scenario_slices",
+                    description=description,
+                    createparents=True,
```

Comment:
Should `createparents=True` also be applied to the "scenario_combinations" table creation on line 468?